NamespaceFixer/source.extension.vsixmanifest

```diff
@@ -1,20 +1,20 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
     <Metadata>
-        <Identity Id="pch9d11e-9ed1-48f6-875b-4d144a5d790c" Version="1.5" Language="en-US" Publisher="pcerv" />
+        <Identity Id="pch9d11e-9ed1-48f6-875b-4d144a5d790c" Version="1.6" Language="en-US" Publisher="pcerv" />
         <DisplayName>NamespaceFixer</DisplayName>
         <Description xml:space="preserve">VS Extension to fix namespaces following C# convention</Description>
         <Icon>Package.ico</Icon>
     </Metadata>
     <Installation>
```

Comment:
There's two `InstallationTarget`'s.  Maybe that's the problem?

I removed the first one and it's working via the debugger in VS 2019

![image](https://user-images.githubusercontent.com/14079228/54634046-bfae6b80-4a57-11e9-9461-00e6866f3119.png)


![image](https://user-images.githubusercontent.com/14079228/54634012-ae655f00-4a57-11e9-8600-57be26a523ec.png)

Reply:
Seems to work 😊

![image](https://user-images.githubusercontent.com/14079228/54636418-cd1a2480-4a5c-11e9-8d69-be02f161747b.png)

[I finally figured out how to create the installer 😊](https://docs.microsoft.com/en-us/visualstudio/extensibility/walkthrough-publishing-a-visual-studio-extension?view=vs-2017#package-your-extension)

Reply:
@pauer24 (not sure if you're notified automatically or not?)